contracts/RewardsHandler.vy

```diff
@@ -354,6 +359,22 @@ def _set_scaling_factor(new_scaling_factor: uint256):
     log ScalingFactorUpdated(new_scaling_factor)
 
 
+@external
+def set_stablecoin_lens(_lens: address):
+    """
+    @notice Setter for the stablecoin lens that determines stablecoin circulating supply.
+    @param _lens The address of the new stablecoin lens.
+    """
+    access_control._check_role(RATE_MANAGER, msg.sender)
+    self._set_stablecoin_lens(IStablecoinLens(_lens))
```

Comment:
I would leave this only to the dao, can we make a separate role? Another solution can be use DEFAULT_ADMIN_ROLE since only the dao has it (need to double check if this works)